instance/static/js/src/openedx_appserver.js

```diff
@@ -111,6 +111,25 @@ app.controller("OpenEdXAppServerDetails", ['$scope', '$state', '$stateParams', '
                               'alert');
             });
         };
+        $scope.$on("websocket:object_log_line", function(event, data){
+            if (!$scope.appserverLogs) {
+                return;
+            }
+
+            if (data.appserver_id == $scope.appserver.id || ($scope.appserver.server && data.server_id == $scope.appserver.server.id)) {
+                if (data.log_entry.level == 'ERROR' || data.log_entry.level == 'CRITICAL') {
```

Comment:
If we keep these filters, can we move the conditional to the backend? This would reduce the traffic and privacy/security risks mentioned earlier.

Reply:
@viadanna, I am not sure I understand your comment here. The filtering here is for segregating the error and non-error messages in separate arrays and I am not sure how it is related to the issue of not restricting access to the websocket notifications.

Reply:
Nevermind, I misunderstood the indentation and blocks here, all's good.